### PR TITLE
Added pdb

### LINKED ISSUE #3118: [Improve Concept Docs]: TRACEBACKS - Add PDB section
If you have not yet contributed to concept documents, this issue will require some upfront reading to give you the needed background knowledge.  Additionally, we recommend reading the existing `about.md` docs [here](https://github.com/exercism/python/blob/main/concepts/functools/about.md), and an example of completed concept docs [here](https://github.com/exercism/python/tree/main/concepts/classes).

## ✅   Getting started

**`Please please read the docs before starting.`** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. 

**General Contributing Docs:**

- [Contributing to Exercism](https://exercism.org/docs/building) | [Exercism and GitHub](https://exercism.org/docs/building/github) | - [Contributor Pull Request Guide](https://exercism.org/docs/building/github/contributors-pull-request-guide)
- [What are those Weird Task Tags about?](https://exercism.org/docs/building/product/tasks)
- [Exercism Formatting and Style Guide](https://exercism.org/docs/building/markdown/style-guide)
- [Exercism Markdown Specification](https://exercism.org/docs/building/markdown/markdown)
- [Reputation](https://exercism.org/docs/using/product/reputation)  

**Documents on Language Tracks and Concepts:**

- [Building Language Tracks: An Overview](https://exercism.org/docs/building/tracks) 
- [What are Concepts?](https://exercism.org/docs/building/tracks/concepts)
- [Concept Specifications](https://exercism.org/docs/building/tracks/concepts) 

<br>

## 🙏🏽   Desired Improvements and Changes

Add section on [PDB](https://realpython.com/python-debugging-pdb/), either to #3059 before it's merged or to the [merged doc](https://github.com/exercism/python/tree/main/docs/TRACEBACKS.md) after the PR is approved and merged.
___________

# 🌟   Original Specifications

Below are the original specifications from the Concept issue.  Please use these as a guide in addition to the change list above.

Learn a little around error tracing tools such as [PDB](https://docs.python.org/3.9/library/pdb.html), and similar tools in common IDEs.

## 🎶  Implementation Notes

- Example code  should **only use syntax & concepts introduced within these docs or one of the prerequisite concept exercises or documents.** Where possible, please use REPL formatting, unless you are demonstrating pseudo code or a long code block.
  Please **do not use** syntax not previously covered in prerequisite topics or exercises.  Please also follow [PEP8](https://www.python.org/dev/peps/pep-0008/) guidelines.
- Our markdown and JSON files are checked against [prettier](https://prettier.io/) .  We recommend [setting prettier up locally](https://prettier.io/docs/en/install.html) and running it prior to submitting your PR  to avoid any CI errors.

<br>

##  🆘  Next Steps & Getting Help

1.  **`If you'd like to work on this issue, comment saying "I'd like to work on this"`** (_there is no real need to wait for a response, just go ahead, we'll assign you and put a `[claimed]` label on the issue_).
2. If you have any questions while implementing, please post the questions as comments in here, or contact one of the maintainers on our Slack channel.

### PR DESCRIPTION
**Pull request by meatball133**
Resolves #3118

### CODE REVIEW

File: docs/TRACEBACKS.md [hunk 1]
```diff
@@ -188,8 +188,117 @@ Setting `PYTHONOPTIMIZE` to `1` is equivalent to running Python with the `-O` op
...
+```
+
+Breakpoints are set up by `break <filename>:<line number> <condition>` where the condition is an optional condition that has to be true for the breakpoint to be hit.
+You can simply write `break` to get a list of the breakpoints you have set.
+To disable a breakpoint you can write `disable <breakpoint number>`.
+To enable a breakpoint you can write `enable <breakpoint number>`.
+To delete a breakpoint you can write `clear <breakpoint number>`.
+To continue execution you can write `continue` or `c`. To exit the debugger you can write `quit` or `q`.
+
+Here is an example of how to use the debugger based on the code earlier:
```
Comment (BethanyG) [inline docs/TRACEBACKS.md:248]:
```suggestion
Here is an example of how to use the above debugger  commands based on the code earlier:
```

File: docs/TRACEBACKS.md [hunk 2]
```diff
@@ -188,8 +188,117 @@ Setting `PYTHONOPTIMIZE` to `1` is equivalent to running Python with the `-O` op
...
+... Num Type         Disp Enb   Where
+... 1   breakpoint   keep no    at c:\pdb.py:4
+...         breakpoint already hit 1 time
+>>> clear break 1
+... Deleted breakpoint 1 at c:\pdb.py:4
+>>> break
+...
+```
+
+In python 3.7 and newer there is an easier way to create breakpoints You can simply write `breakpoint()` where you want to create a breakpoint.
```
Comment (BethanyG) [inline docs/TRACEBACKS.md:277]:
```suggestion
In Python 3.7+  there is an easier way to create breakpoints 
Simply writing `breakpoint()` where needed will create one.
```

**Review by BethanyG — APPROVED** [review]
Hi @meatball133 👋🏽 

Thank you again for submitting this PR.  My sincere apologies that it took so long to review.  This all looks good, and I've approved! 🎉 🌟 

I've also left a few suggestions, which you can take or leave.  Once you're done, either you or I can merge it and send it on its happy way to production.  😄

**Comment by meatball133** [comment]
I feel happy about it. I don't have the permissions to merge it so you will have to do it.

**Pull request merged**